test/Engine/ProtoTest/LiveRunnerTests/MemoryConsumptionTests.cs

```diff
@@ -0,0 +1,71 @@
+﻿﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using NUnit.Framework;
+using ProtoCore.DSASM.Mirror;
+using ProtoCore.Lang;
+using ProtoTest.TD;
+using ProtoScript.Runners;
+using ProtoTestFx.TD;
+using System.Linq;
+using ProtoCore.AST.AssociativeAST;
+using ProtoCore.DSASM;
+using ProtoCore.Mirror;
+using System.Collections;
+
+namespace ProtoTest.LiveRunner
+{
+    public class MemoryConsumptionTests : ProtoTestBase
+    {
+        private int instrStreamStart = 0;
+        private int instrStreamEnd = 0;
+        ProtoCore.Diagnostics.Runtime runtimeDiagnostics = null;
+        private ProtoScript.Runners.LiveRunner liverunner = null;
+
+        public override void Setup()
+        {
+            base.Setup();
+            liverunner = new ProtoScript.Runners.LiveRunner(); 
+            runtimeDiagnostics = new ProtoCore.Diagnostics.Runtime(liverunner.RuntimeCore);
+        }
+
+        public override void TearDown()
+        {
+            base.TearDown();
+            liverunner.Dispose();
```

Comment:
Although it is just for testing, but I think for initialization, initialize base's stuff firstly. For disposing, in reverse order.